frontend-next-migration/src/shared/ui/CustomForm/ui/CustomForm.tsx

```diff
@@ -132,13 +136,107 @@ function Button({ children, className = '', ...props }: ButtonProps) {
     );
 }
 
+/**
+ * MultiSelectionDropdown component for rendering a multi-selection dropdown.
```

Comment:
Example code works. Personally I would have added more props that `MultiSelectionDropdown` uses, like `disableSearch`.